Provide Clock.Until

https://golang.org/pkg/time/#Until

Hm, tests are currently failing:
```
tereshkova ~/p/clock [master] $ go test -v .
=== RUN   TestClock_After
--- PASS: TestClock_After (0.02s)
=== RUN   TestClock_AfterFunc
panic: Fail in goroutine after TestClock_After has completed

goroutine 8 [running]:
testing.(*common).Fail(0xc0001321a0)
        /home/dustin/.gvm/gos/go1.17/src/testing/testing.go:710 +0xe5
testing.(*common).FailNow(0xc0001321a0)
        /home/dustin/.gvm/gos/go1.17/src/testing/testing.go:732 +0x1e
testing.(*common).Fatal(0x1c9c380, {0xc0000577c0, 0x0, 0x0})
        /home/dustin/.gvm/gos/go1.17/src/testing/testing.go:824 +0x4f
github.com/benbjohnson/clock.TestClock_After.func2()
        /home/dustin/p/clock/clock_test.go:21 +0x5e
created by github.com/benbjohnson/clock.TestClock_After
        /home/dustin/p/clock/clock_test.go:19 +0xb2
FAIL    github.com/benbjohnson/clock    0.034s
FAIL
```

--- a/clock.go
+++ b/clock.go
@@ -167,12 +167,12 @@ func (m *Mock) Now() time.Time {
 	return m.now
 }
 
-// Since returns time since t, using the mock clock's wall time.
+// Since returns time since `t` using the mock clock's wall time.
 func (m *Mock) Since(t time.Time) time.Duration {
 	return m.Now().Sub(t)
 }
 
-// Until returns time until t, using the mock clock's wall time.
+// Until returns time until `t` using the mock clock's wall time.
 func (m *Mock) Until(t time.Time) time.Duration {
 	return t.Sub(m.Now())
 }

--- a/clock_test.go
+++ b/clock_test.go
@@ -300,11 +300,11 @@ func TestMock_Until(t *testing.T) {
 
 	end := clock.Now().Add(500 * time.Second)
 	if dur := clock.Until(end); dur.Seconds() != 500 {
-		t.Fatalf("expected 500 to the end, actually: %v", dur.Seconds())
+		t.Fatalf("expected 500s duration between `clock` and `end`, actually: %v", dur.Seconds())
 	}
 	clock.Add(100 * time.Second)
 	if dur := clock.Until(end); dur.Seconds() != 400 {
-		t.Fatalf("expected 400 to the end, actually: %v", dur.Seconds())
+		t.Fatalf("expected 400s duration between `clock` and `end`, actually: %v", dur.Seconds())
 	}
 }
 

